libs/note-manager/src/note-manager.ts

```diff
@@ -69,7 +70,7 @@ export class NoteManager {
     version: 'v1',
     width: '5',
     index: 0,
-  };
+  } as const satisfies DefaultNoteGenInput;
```

Comment:
Explain?

Reply:
I have made these changes by adding `as const` here to indicate that all the fields of this object are `readonly` (immutable) fields, as they represent default values for generating the note (Read more [here](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions)). The statement `satisfies DefaultNoteGenInput` is used for TypeScript type checking, ensuring that only properties defined in the `DefaultNoteGenInput` interface are allowed (Read more [here](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-9.html#the-satisfies-operator)).